packages/react-dom/src/__tests__/utils/ReactDOMServerIntegrationTestUtils.js

```diff
@@ -89,7 +89,7 @@ module.exports = function(initModules) {
         console.log(
           `We expected ${count} warning(s), but saw ${filteredWarnings.length} warning(s).`,
         );
-        if (filteredWarnings.count > 0) {
+        if (filteredWarnings.length > 0) {
```

Comment:
Drive-by fix that was required to get hydration mismatch warnings during development. This adds some noise to this test since there are a bunch of tests that expect more warnings than there actually are. 

I started working on them by switching to `toErrorDev` instead and it's a lot of work to fully switch. Though it revealed some odd error messages and that you also hit https://github.com/facebook/react/pull/22797 in your tests. So definitely worth pursuing IMO.